src/layer/vector/Circle.js

```diff
@@ -43,8 +43,13 @@ export var Circle = CircleMarker.extend({
 	// @method setRadius(radius: Number): this
 	// Sets the radius of a circle. Units are in meters.
 	setRadius: function (radius) {
+		var oldRadius = this._mRadius;
 		this._mRadius = radius;
-		return this.redraw();
+		this.redraw();
+
+		// @event move: Event
+		// Fired when the circle radius is changed via [`setRadius`](#circle-setradius). Old and new radius are included in event arguments as `oldRadius`, `radius`.
+		return this.fire('move', {oldRadius: oldRadius, radius: this._mRadius});
```

Comment:
For sake of clarity it may be better to duplicate `setLatLngs` here too.
Not sure though.

Reply:
Also, this may be a (a bit of) breaking change, as some code may expect latlng properties to always be included..

Reply:
It is already different between marker / polyline because of `latlng` and `latlngs`, so I think it needs to be handled for each shape different, then the user can do by him self `getLatLng()`. But I added the property `latlng` (it doesn't hurt if it is there too)